From 1748d59460ade1baeb93de4afee55f0989f7510b Mon Sep 17 00:00:00 2001
From: Paul Thomson <paulthomson@google.com>
Date: Fri, 17 Apr 2026 13:06:41 +0100
Subject: [PATCH 03/12] Remove some FE10 compiler backends

This change appears to exclude some compiler backends (js, wasm) that
are presumably not used by Lint. I assume this is to save space.
---
 .../kotlin-compiler-fe10-for-ide/build.gradle.kts            | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/prepare/ide-plugin-dependencies/kotlin-compiler-fe10-for-ide/build.gradle.kts b/prepare/ide-plugin-dependencies/kotlin-compiler-fe10-for-ide/build.gradle.kts
index aeef202c737..268417a8aff 100644
--- a/prepare/ide-plugin-dependencies/kotlin-compiler-fe10-for-ide/build.gradle.kts
+++ b/prepare/ide-plugin-dependencies/kotlin-compiler-fe10-for-ide/build.gradle.kts
@@ -8,7 +8,10 @@ val excludedCompilerModules = listOf(
     ":compiler:cli",
     ":compiler:cli-js",
     ":compiler:javac-wrapper",
-    ":compiler:incremental-compilation-impl"
+    ":compiler:incremental-compilation-impl",
+    ":compiler:backend.js",
+    ":compiler:backend.wasm",
+    ":js:js.dce",
 )
 
 val extraCompilerModules = listOf(
-- 
2.54.0.rc1.513.gad8abe7a5a-goog

